src/p2p/net_node.cpp

```diff
@@ -55,6 +125,8 @@ namespace nodetool
     const command_line::arg_descriptor<std::vector<std::string> > arg_p2p_add_exclusive_node   = {"add-exclusive-node", "Specify list of peers to connect to only."
                                                                                                   " If this option is given the options add-priority-node and seed-node are ignored"};
     const command_line::arg_descriptor<std::vector<std::string> > arg_p2p_seed_node   = {"seed-node", "Connect to a node to retrieve peer addresses, and disconnect"};
+    const command_line::arg_descriptor<std::vector<std::string> > arg_proxy = {"proxy", "<network-type>,<socks-ip:port>[,max_connections] i.e. \"tor,127.0.0.1:9050,100\""};
+    const command_line::arg_descriptor<std::vector<std::string> > arg_anonymous_inbound = {"anonymous-inbound", "<hidden-service-address>,<bind-ip:port>[,max_connections] i.e. \"tor,x.onion,100\""};
```

Comment:
I'm guessing "tor" is not a valid hidden service address. I've never set one up. It made sense as network-type. Also, you probably want <bind-ip>[:<port>] since port seems optional as per the example.

Reply:
I caught the Tor typo in the last round of edits, and will update the port thing too.